etc/deploy/dd/dd_notebook_base.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jupyter-notebook
+  labels:
+    app: jupyter-notebook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jupyter-notebook
+  template:
+    metadata:
+      labels:
+        app: jupyter-notebook
+    spec:
+      containers:
+        - name: pach-notebook
+          image: pachyderm/notebooks-user:vPACH_VERSION
```

Comment:
should this change `vPACH_VERSION -> "v${PACH_VERSION}"`?

Reply:
Naw, it's just a place holder, the `sed` replaces it at release time.